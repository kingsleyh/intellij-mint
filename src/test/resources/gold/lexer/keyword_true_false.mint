module A {

  fun woop() : Bool {
    if(true) {
      false
    } else {
      true
    }
  }
}