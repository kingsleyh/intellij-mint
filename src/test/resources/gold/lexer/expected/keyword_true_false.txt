module ('module')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('woop')
( ('(')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Bool')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
if ('if')
( ('(')
true ('true')
) (')')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
false ('false')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE (' ')
else ('else')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
true ('true')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n')
} ('}')